packages/outline-view/src/browser/outline-view-tree-model.ts

```diff
@@ -39,4 +39,14 @@ export class OutlineViewTreeModel extends TreeModelImpl {
         }
         return false;
     }
+
+    /**
+     * The default behavior of `doOpenNode` is to toggle the expansion of the node.
+     * Override this behavior in order to prevent expansion, but allow for the
+     * `onOpenNode` event to still fire on a double-click event.
```

Comment:
The mention of the event here is a little confusing, since the event was never fired from this method and clearly isn't being fired in this no-op override. As I understand it, you're saying that you chose to override this method rather than `openNode` in order to allow the event - which is fired in *that* method - to fire. One alternative would be to override `openNode` and omit the call to `doOpenNode` - then your decision to fire the event without doing anything else would be explicit.